src/components/05_pages/Content/Content.js

```diff
@@ -184,288 +186,308 @@ class Content extends Component {
-                          text="This action cannot be undone."
-                          cancelText="Cancel"
-                          confirmText="Delete"
-                          enterAction={() => {
-                            this.props.contentDelete(node);
-                          }}
-                        >
-                          <DeleteIcon />
-                        </OpsModalButton>
-                      </TableCell>
-                    </TableRow>
-                  );
-                })}
-              </TableBody>
-            </Table>
-            <TablePagination
-              component="div"
-              count={count}
-              rowsPerPage={limit}
-              page={offset / limit}
-              onChangePage={this.pageChangeHandler}
-              rowsPerPageOptions={[limit]}
-              labelDisplayedRows={({ page }) => `Page: ${page + 1}`}
-              nextIconButtonProps={{ 'aria-label': 'Next content page.' }}
-              backIconButtonProps={{ 'aria-label': 'Previous content page.' }}
-            />
-            {!this.props.contentList.length && (
-              <Typography className={styles.noContentMessage}>
-                There is no content yet. {<Link to="/node/add">Add one</Link>}.
-              </Typography>
-            )}
-          </div>
+                              <EditIcon />
+                            </IconButton>
+                            <OpsModalButton
+                              aria-label="delete"
+                              className={styles.button}
+                              title={`Are you sure that you want to delete this content ${title}?`}
+                              text="This action cannot be undone."
+                              cancelText="Cancel"
+                              confirmText="Delete"
+                              enterAction={() => {
+                                this.props.contentDelete(node);
+                              }}
+                            >
+                              <DeleteIcon />
+                            </OpsModalButton>
+                          </TableCell>
+                        </TableRow>
+                      );
+                    })}
+                  </TableBody>
+                </Table>
+                <TablePagination
+                  component="div"
+                  count={count}
+                  rowsPerPage={limit}
+                  page={offset / limit}
+                  onChangePage={this.pageChangeHandler}
+                  rowsPerPageOptions={[limit]}
+                  labelDisplayedRows={({ page }) => `Page: ${page + 1}`}
+                  nextIconButtonProps={{ 'aria-label': 'Next content page.' }}
+                  backIconButtonProps={{
+                    'aria-label': 'Previous content page.',
+                  }}
+                />
+                {!this.props.contentList.length && (
```

Comment:
Mh, this condition wouldn't be triggered anymore now. Should we remove it?

Reply:
I think I refined this a bit in #5243068

Reply:
👍